[2025-05-24T22:58:04.291+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-24T22:58:04.321+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: supplier_data_etl_pipeline.upload_merged_to_s3 manual__2025-05-24T22:54:15.604677+00:00 [queued]>
[2025-05-24T22:58:04.327+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: supplier_data_etl_pipeline.upload_merged_to_s3 manual__2025-05-24T22:54:15.604677+00:00 [queued]>
[2025-05-24T22:58:04.327+0000] {taskinstance.py:2303} INFO - Starting attempt 3 of 8
[2025-05-24T22:58:04.338+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): upload_merged_to_s3> on 2025-05-24 22:54:15.604677+00:00
[2025-05-24T22:58:04.367+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2236) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-24T22:58:04.371+0000] {standard_task_runner.py:63} INFO - Started process 2240 to run task
[2025-05-24T22:58:04.372+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'supplier_data_etl_pipeline', 'upload_merged_to_s3', 'manual__2025-05-24T22:54:15.604677+00:00', '--job-id', '184', '--raw', '--subdir', 'DAGS_FOLDER/supplier_dag.py', '--cfg-path', '/tmp/tmpx2jz3qna']
[2025-05-24T22:58:04.373+0000] {standard_task_runner.py:91} INFO - Job 184: Subtask upload_merged_to_s3
[2025-05-24T22:58:04.436+0000] {task_command.py:426} INFO - Running <TaskInstance: supplier_data_etl_pipeline.upload_merged_to_s3 manual__2025-05-24T22:54:15.604677+00:00 [running]> on host a7d211f7b093
[2025-05-24T22:58:04.542+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='supplier_data_etl_pipeline' AIRFLOW_CTX_TASK_ID='upload_merged_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2025-05-24T22:54:15.604677+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-24T22:54:15.604677+00:00'
[2025-05-24T22:58:04.544+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-24T22:58:05.997+0000] {etl.py:159} INFO - Successfully uploaded to s3://fmcg-de-assessment/supplier_data/merged_supplier_data.csv
[2025-05-24T22:58:06.000+0000] {python.py:237} INFO - Done. Returned value was: True
[2025-05-24T22:58:06.004+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-24T22:58:06.025+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=supplier_data_etl_pipeline, task_id=upload_merged_to_s3, execution_date=20250524T225415, start_date=20250524T225804, end_date=20250524T225806
[2025-05-24T22:58:06.052+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-24T22:58:06.073+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-24T22:58:06.074+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
